src/esp/sim/Simulator.cpp

```diff
@@ -39,6 +42,14 @@ using metadata::attributes::SceneAOInstanceAttributes;
 using metadata::attributes::SceneObjectInstanceAttributes;
 using metadata::attributes::StageAttributes;
 
+namespace {
+constexpr const char* shadowMapDrawableGroupName = "static-shadow-map";
+constexpr const char* defaultRenderingGroupName = "";
+const int shadowMapSize = 1024;
+const int maxNumShadowMaps = 3;  // the max number of point shadow maps
```

Comment:
Yeah, I admit we did not strictly follow every item in the FB C++ guide.
For this one, adding `k` or `g` to the variables will make things complicated. My opinion is to avoid it. :)